bin/fba_sv1

```diff
@@ -78,24 +78,78 @@ def get_isaen(g, aen):
     )
 
 
-# AR get the updated ra,dec with proper motion
-def get_nowradec(ra, dec, pmra, pmdec, parallax, ref_epoch, new_obstime):
+# AR courtesy of DL : from astrometry.net's util/starutil_numpy.py
+def datetomjd(d):
+    d0 = datetime(1858, 11, 17, 0, 0, 0)
+    dt = d - d0
+    return dt.days + (dt.seconds + dt.microseconds / 1e6) / 86400.0
```

Comment:
You might compare the results of this with astropy.time.Time(date).mjd.  I hadn't used apply_space_motion before, but I have used the astropy.time library a lot, and would expect this to work and be more obvious.

Reply:
correct; besides the datetomjd() is just used once in "mjd2k = datetomjd(J2000)".